lib/hanami/environment.rb

```diff
@@ -487,10 +492,11 @@ def set_hanami_env_vars!
     # @since 0.2.0
     # @api private
     def set_application_env_vars!
-      dotenv = root.join(DEFAULT_DOTENV_ENV % environment)
-      return unless dotenv.exist?
-
-      env.load!(dotenv)
+      DEFAULT_DOTENV_ENV_PATTERNS.each do |filename_format|
+        next if filename_format == '.env.local' && environment == 'test'
```

Comment:
@gmanley why `.env.local` isn't loaded if env is test?

Reply:
@jodosha I was trying to match behavior with dotenv rails. If you look at the chart: https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use `.env.local` is ignored in test. I assume the rational is you don't want real api keys set. I can change this, though it may be confusing to have different behavior between dotenv-rails and hanami. That is kind of what I was trying to get away from.